imports/plugins/included/product-detail-simple/client/containers/variantListContainer.js

```diff
@@ -187,7 +204,7 @@ function composer(props, onData) {
   }
 
   let editable;
-
+  const products = props.product;
```

Comment:
Does this exist in this scope? It belong to else block and you don't really need to cache it.